src/include/wiredtiger.in

```diff
@@ -1794,6 +1794,20 @@ struct __wt_session {
 	 * @config{round_to_oldest, if read timestamp is earlier than oldest
```

Comment:
I think, we no longer support this "round_to_oldest" parameter. So, we need to remove it.

Reply:
@smani87, that will come in a later commit so we don't have to update WT in master and make the SERVER change to use the new API atomically.